components/embedder_traits/resources.rs

```diff
@@ -49,6 +44,25 @@ pub fn sandbox_access_files_dirs() -> Vec<PathBuf> {
         .sandbox_access_files_dirs()
 }
 
+pub fn filename(file: Resource) -> &'static str {
```

Comment:
Perhaps this could be a function on the enum itself:

```rust
impl Resource {
    pub fn filename(&self) -> &'static str {
        ....
    }
}

Reply:
done in 73c665011e20accdd7c4af0833f4e15086aaa88f